### PR TITLE
added package version resolutions

### PR DESCRIPTION
**Pull request by shunjizhan**
## Change
When I did a fresh clone and `yarn install`, there are lots of package version mismatch, and so tests were not able to run. To solve this and prevent future issue, I added package resolutions and lock file, so package version is in line with `bodhi.js` https://github.com/AcalaNetwork/bodhi.js/blob/master/common/config/rush/.pnpmfile.cjs#L21-L31

Also, after some experiment, I believe for all state changing calls, we will need to specify gas params. For examples, for Echo tests, before adding gas params it will throw bad gas error, and after adding these params, `yarn test-mandala` now all passed.

## Test
`yarn test-mandala` 
- passed for `hello-world` and `echo` after adding gas params. 
- not passed yet for `NFT` and `token` since they don't have gas params yet

## TODO
- we will need to add gas params for all state-changing calls in `NFT` and `token` tests

### CODE REVIEW

**Comment by shunjizhan** [comment]
Actually, we might be able to omit these gas params in the future.

One example error I got before fixing gas is 
```
ProviderError: substrate gasLimit, gasPrice, storageLimit should all be greater than 0 (txGasLimit="89914", txGasPrice="200000471050", maxPriorityFeePerGas=undefined, maxFeePerGas=undefined, txFeePerGas="199999946752", storageDepositPerByte="100000000000000", gasLimit="-230086", validUntil="240", storageLimit="640", code=INVALID_ARGUMENT, version=bodhi.js/providers/2.1.7)
```
It uses `txGasLimit="89914", txGasPrice="200000471050"` which are from `eth_gasPrice()` and `eth_estimateGas()` endpoints. It SHOULD work but aren't working now https://github.com/AcalaNetwork/bodhi.js/issues/209

**Comment by ThunderDeliverer** [comment]
Ok, if that's the case, then I think we shouldn't merge this one and remove the forcing of the gas parameters from the tutorials. The tutorials are supposed to be instructions on how to use the Acala EVM+ and I don't think we should take shortcuts with them. If they are not working at the moment, we have to make sure that they start working and only then will the tutorials as well as the EVM+ be ready to be released. 🤔 

We can discuss it further in today's planning meeting.

File: echo/test/Echo.js [hunk 1]
```diff
@@ -34,23 +34,35 @@ describe("Echo contract", async function () {
                 this.timeout(50000);
 
                 it("should update the echo variable", async function () {
-                        await instance.scream("Hello World!");
+                        await instance.scream("Hello World!", {
+                                gasPrice: ethParams.txGasPrice,
+                                gasLimit: ethParams.txGasLimit
+                        });
```
Comment (ThunderDeliverer) [inline echo/test/Echo.js:40]: Changes like this warrant changes in the respective `README` files. We should finalize our decision on how to proceed and then either remove the gas parameters or update the `READMEs `as well as the scripts.

**Comment by shunjizhan** [comment]
sounds great, let's wait til the gasprice problem is fixed, and see what we will do from there

**Review by ThunderDeliverer — APPROVED** [review]
As we agreed on the planning meeting, the gas transaction parameters were removed from this PR. Everything else seems good to me 👌

**Pull request merged**